Modules/Services/Sources/Services/BlockService/BlockService.swift

```diff
@@ -26,6 +26,11 @@ public final class BlockService: BlockServiceProtocol {
         return response.blockID
     }
     
+    public func addFirstBlock(contextId: String, info: BlockInformation) async throws -> String {
+        let headerBlockId = "header"
+        return try await add(contextId: contextId, targetId: headerBlockId, info: info, position: .bottom)
```

Comment:
We have similar logic in SetObjectCreationHelper. Try to search EditorConstants.headerBlockId